Readme.md

```diff
@@ -1,155 +1,135 @@
-be all you need. You can update the `rustc_codegen_gcc` without worrying about GCC.
-
-### Building with your own GCC version
-
-If you wrote a patch for GCC and want to test it without this backend, you will need
-to do a few more things.
+#### Other architectures
+If you are on a host arch other than x86-64 Linux or are targetting an arch other than x86-64 Linux, you will need to build a custom GCC with `libgccjit.so`. **At this time, this requires the use of the [rust-lang/gcc](https://github.com/rust-lang/gcc) fork of GCC, which includes patches to libgccjit to enable the use of this codegen.**
 
-To build it (most of these instructions come from [here](https://gcc.gnu.org/onlinedocs/jit/internals/index.html), so don't hesitate to take a look there if you encounter an issue):
+Full instructions to build libgccjit are provided in the [libgccjit documentation](https://gcc.gnu.org/onlinedocs/jit/internals/index.html), so check there if you encounter issues, however brief directions for Debian-based Linux follow below. You may need to adapt them to your operating system and package manager. If you need to build a cross-compiler, see [Building a cross-compiler with rustc_codegen_gcc](./doc/cross.md).
 
 ```bash
-$ git clone https://github.com/antoyo/gcc
+$ git clone https://github.com/rust-lang/gcc gcc-source
 $ sudo apt install flex libmpfr-dev libgmp-dev libmpc3 libmpc-dev
 $ mkdir gcc-build gcc-install
 $ cd gcc-build
-$ ../gcc/configure \
+$ ../gcc-source/configure \
     --enable-host-shared \
     --enable-languages=jit \
-    --enable-checking=release \ # it enables extra checks which allow to find bugs
+    --enable-checking=release \
     --disable-bootstrap \
     --disable-multilib \
     --prefix=$(pwd)/../gcc-install
-$ make -j4 # You can replace `4` with another number depending on how many cores you have.
+$ make -j4
+$ make install
 ```
 
-If you want to run libgccjit tests, you will need to also enable the C++ language in the `configure`:
+Notes:
+* If you want to run libgccjit tests, you must also add C++ to the enabled languages: `--enable-languages=jit,c++`
+* `--enable-host-shared` builds the compiler as position independent code and is required to build libgccjit. This results in a slower compiler; however, if building GCC in multiple passes, `jit` may be built in the first pass with `--enable-host-shared`, with both disabled in subsequent passes.
+* `--enable-checking=release` and `--disable-bootstrap` speed up compilation by disabling self-checks and may be omitted.
+* `--disable-multilib` is used as libgccjit only supports targeting one arch ABI variant.
+* Adjust the `4` in `make -j4` to the number of threads available on your system to speed up compilation.
+* Change `make install` to `make install-strip` to remove debug symbols from GCC, including `libgccjit.so`, to save on disk space.
 
+Once the build is complete, one may run libgccjit tests like so (requires DejaGnu to be installed):
```

Comment:
No it's not needed.

Reply:
As I mentioned elsewhere, it is required and some systems don't have it installed by default.